python/tests/test_jit.py

```diff
@@ -133,6 +133,91 @@ def coalescent_nodes_python(ts):
     np.testing.assert_array_equal(C1, C2)
 
 
+@pytest.mark.parametrize("ts", tsutil.get_example_tree_sequences())
+def test_jit_diversity(ts):
+    if ts.num_samples < 1:
+        pytest.skip(
+            "Tree sequence must have at least one sample for diversity calculation"
+        )
+
+    import tskit.jit.numba as jit_numba
+
+    NODE_IS_SAMPLE = tskit.NODE_IS_SAMPLE
+
+    @numba.njit
```

Comment:
Why are importing the jit module within the test functions here and defining the algorithm? I think we can assume that developers have numba installed, and there's pytest ways of skipping the module for CI?